tests/robot-tests/tests/admin/analyst/role_ui_permissions/publication_approver_ui_permissions.robot

```diff
@@ -14,19 +14,24 @@ Force Tags          Admin    Local    Dev
 *** Test Cases ***
 Import permissions test variables
     Import bootstrap data roles and permissions variables
-    Set suite variable    ${PUBLICATION_NAME}    ${PUBLICATION_FOR_RELEASE_APPROVER}
+    Set suite variable    ${PUBLICATION_NAME}    ${PUBLICATION_FOR_PUBLICATION_APPROVER}
 
-Validates release approver publication page is correct
+Validates publication approver publication page is correct
     user navigates to publication page from dashboard    ${PUBLICATION_NAME}
 
     user waits until page contains link    Releases
     user waits until page contains link    Methodologies
-    user waits until page contains link    Legacy releases    # remove as part of EES-3794
```

Comment:
Just wondering why we're removing the `EES-3794` comment here and throughout this PR?

Reply:
I only realised that the [EES-3794](https://dfedigital.atlassian.net/browse/EES-3794) ticket existed to hide the 'Legacy releases' tab when I saw a reference to this in code while updating UI tests.

The result of my change is that contributors can see legacy releases in the dashboard in a read-only view of them.

I mentioned this to @Cam Race today and he said a read-only view of legacy releases is fine for now.

I've left a comment on EES-3794 to explain and left it open in case we still want to go ahead with removing it completely in future.

This PR's primary change (hiding Create release/methodology buttons for archived publications) has nothing to do with legacy releases, so what started out intending to be a small bugfix while working in the same area of dashboard permissions eventually turned out to be a bigger change than expected. 😢

With hindsight I should have stopped and made it into it's own change, in which case I'd have probably spotted EES-3794 earlier!